setup.py

```diff
@@ -22,6 +22,13 @@ def read(*filenames, **kwargs):
     exec(file.read(), about)
 
```

Comment:
I think we need to polyfill https://pypi.python.org/pypi/flake8-polyfill#retrieving-standard-in